UI/obs-app.cpp

```diff
@@ -2965,51 +2965,6 @@ static bool update_reconnect(ConfigFile &config)
 	return false;
 }
 
-static void convert_x264_settings(obs_data_t *data)
-{
-	bool use_bufsize = obs_data_get_bool(data, "use_bufsize");
-
-	if (use_bufsize) {
-		int buffer_size = (int)obs_data_get_int(data, "buffer_size");
-		if (buffer_size == 0)
-			obs_data_set_string(data, "rate_control", "CRF");
-	}
-}
-
-static void convert_14_2_encoder_setting(const char *encoder, const char *file)
```

Comment:
This function was replaced with `convert_28_1_encoder_setting` by d7c7180a8c so that this became unused.

Reply:
I wonder why we did that.

Reply:
The function `convert_14_2_encoder_setting` was added by commit 1f8fc8e8ef0.
Do you think we still need to call `convert_14_2_encoder_setting` so that a config file from 14.2 or older can still be correctly read?

Reply:
I think at this point it's reasonable for us to remove ancient migrations such as this. tbh I don't like how migrations work at all and would prefer we rethink them at some point.